commands/gstatus.py

```
@@ -0,0 +1,408 @@
+    print("")
+    
+    # Get repository and PR information
+    owner, repo = get_repo_info()
+    if not owner or not repo:
+        print("❌ **Error**: Could not determine repository information")
+        print("Make sure you're in a git repository with GitHub remote")
+        return 1
+    
+    pr_number, pr_url = get_current_pr()
+    if not pr_number:
+        print(f"❌ **No PR found** for current branch")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)